modules/augment/src/rotate.cpp

```diff
@@ -0,0 +1,40 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "precomp.hpp"
+namespace cv {
+namespace augment {
+
+    Rotate::Rotate(Range _angleRange)
+    {
+        this->angleRange = _angleRange;
+        this->resetRandom();
```

Comment:
Please do not use `this` if it's not needed.

Reply:
done